src/AqueductV1Pair.sol

```diff
@@ -816,22 +846,20 @@ contract AqueductV1Pair is IAqueductV1Pair, AqueductV1ERC20, SuperAppBase {
         // set user starting cumulative
         (address user, ) = abi.decode(_agreementData, (address, address));
         if (address(_superToken) == _token0) {
-            uint256 balance1 = UQ112x112.decode(
+            uint256 balance1 = UQ160x96.decode(
                 uint256(uint96(flow0)) * (twap1CumulativeLast - userStartingCumulatives1[user])
             );
 
-            if (balance1 > 0) _safeTransfer(_token1, user, balance1);
+            if (balance1 > 0) userBalances1[user] += balance1;
```

Comment:
Thoughts on a comment to explain why we use this mapping instead of safeTransfer?

Reply:
agreed that would be helpful, added in 564ec07